packages/db-migrations/databases/fxa_oauth/patches/patch-032-031.sql

```diff
@@ -0,0 +1,3 @@
+-- ALTER TABLE `clients` CHANGE COLUMN `redirectUri` VARCHAR(256) NULL DEFAULT NULL;
```

Comment:
Why does the forward migration have `redirectUri` twice, and the reverse migration only once?

Reply:
Typo, fixed